src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogHttpClient.java

```diff
@@ -588,4 +652,9 @@ private String getJenkinsVersion(){
         return this.jenkinsVersion;
     }
 
+    private static boolean isCollectBuildLogEnabled(){
+        return DatadogUtilities.getDatadogGlobalDescriptor() != null &&
+                DatadogUtilities.getDatadogGlobalDescriptor().isCollectBuildLogs();
```

Comment:
Your doing a lookup twice 😓 and your using an more expensive get in the method called.

You should use.
```java
ExtensionList.lookupSingleton(DatadogGlobalConfiguration.class);
```
Also the method could be moved to the `DatadogGlobalConfiguration` class basically moving it to where it belongs.
See here for example: 
https://github.com/jenkinsci/azure-keyvault-plugin/blob/ef3a5eb4413f7ecd47d6532735604c52fceaaef0/src/main/java/org/jenkinsci/plugins/azurekeyvaultplugin/AzureKeyVaultGlobalConfiguration.java#L241-L243

Reply:
That sounds like a good idea, so we should refactor `getDatadogGlobalDescriptor()` and move it to `DatadogGlobalConfiguration`?

Reply:
There is definitely an opportunity to clean this up, but let's cover it in a separate PR.

Reply:
@sarah-witt I see you never picked up this suggestion 😅

Reply:
Some changes were implemented [here](https://github.com/jenkinsci/datadog-plugin/pull/85/files)

Reply:
Ah you kept `DatadogUtilities.getDatadogGlobalDescriptor` 😅 Logically I would prefer `DatadogGlobalConfiguration.get()`